test/service/test_app_logging.py

```diff
@@ -3,15 +3,22 @@
     DEBUG,
     INFO,
 )
+from unittest.mock import (
+    PropertyMock,
+    patch as mock_patch,
```

Comment:
```suggestion
    patch,
```

What's wrong with `patch`?

Reply:
I felt compelled to denote the mock relationship to patch.

Reply:
We don't do that anywhere else and I see no good reason to start now.